wicket-core/src/main/java/org/apache/wicket/markup/html/debug/PageViewCSSResourceReference.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.wicket.markup.html.debug;
+
+import org.apache.wicket.request.resource.CssResourceReference;
+
+public final class PageViewCSSResourceReference extends CssResourceReference
```

Comment:
Could you please write a short javadoc describing the purpose of this class. I see that it's just a style for PageView, but it took some time to figure out.

Reply:
Also why does it need a separate class? Not just a new CssResourceReference(EnhancedPageView.class, "enhancedpageview.css"). Sorry it's not quite clear for me.

Reply:
The class is not really needed, but making it a singleton is more efficient.

Reply:
I've added a short description to the class.

Reply:
`CssResourceReference(PageViewCSSResourceReference.class, "pageview.css")` is just fine. The ResourceRefereces are cached anyway